order/models.py

```diff
@@ -0,0 +1,65 @@
+from django.db import models
+
+from product.models import Product, Product_size
+from account.models import User
+
+class Order_address(models.Model):
```

Comment:
```suggestion
class Address(models.Model):
```
그냥 Address가 나을것 같은데요~!